server/routes/settings/tvdb.ts

```diff
@@ -0,0 +1,50 @@
+import Tvdb from '@server/api/indexer/tvdb';
+import type { TvdbSettings } from '@server/lib/settings';
+import { getSettings } from '@server/lib/settings';
+import logger from '@server/logger';
+import { Router } from 'express';
+
+const tvdbRoutes = Router();
+
+tvdbRoutes.get('/', (_req, res) => {
+  const settings = getSettings();
+
+  res.status(200).json(settings.tvdb);
+});
+
+tvdbRoutes.put('/', (req, res) => {
+  const settings = getSettings();
+
+  if (!settings.tvdb) {
+    settings.tvdb = {
+      use: false,
+    };
+  }
```

Comment:
cf. my previous comment, this `tvdb: { use: boolean }` makes things more complicated.

Reply:
I've replaced the value in settings with a boolean